docs/Support/faq.md

```diff
@@ -66,31 +146,109 @@ Yes! See [boolean logic](/accessControl/conditionTypes/booleanLogic) for example
 
 No, you cannot. If the condition was saved with `"permanent": false` then you can edit the condition associated with the resourceId, but you cannot ever edit the resourceId itself. If you wanted to edit a resourceId, you could create a new one with the same conditions, and as long as the old condition was stored with `"permanent": false` then you could remove all the conditions and make that old resourceId impossible to use.
 
-
 ### 7. Are encryptedSymmetricKey and encryptedString unique to a user or global for a piece of content that we've encrypted?
 
-Global for the piece of content. The only exception, where something is scoped to a user, is if you want to update the access control conditions, because you set permanent: false
-when you stored the access control condition.  In that case, then the wallet address that stored the condition is able to update it, and only that wallet address can update it.
+Global for the piece of content. The only exception, where something is scoped to a user, is if you want to update the access control conditions, because you set permanent: false when you stored the access control condition.  In that case, then the wallet address that stored the condition is able to update it, and only that wallet address can update it.
```

Comment:
```suggestion
Global for the piece of content. The only exception is when something is scoped to a user and you want to update the access control conditions, because you set permanent: false at the time of storing the ACC.  In that case, then the wallet address that stored the condition is able to update it, and only that wallet address can update it.
```